channel/consensus_channel/consensus_channel.go

```diff
@@ -109,6 +114,7 @@ type LedgerOutcome struct {
 	guarantees   map[types.Destination]Guarantee
 }
 
+// includes returns true when o includes g in its list of guarantees.
```

Comment:
Nit: I don't believe `o` makes it into the generated documentation / hover modal in VSCode. So I prefer referencing "the receiver" instead of using the named variable.
<img width="589" alt="Screenshot 2022-03-28 at 20 02 56" src="https://user-images.githubusercontent.com/1833419/160468445-dc83d55e-fcf2-4968-89c9-890a33e75441.png">

Reply:
5c3c1dbac1106b009eeec84d29194d67db66b57f